website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
Interesting! Looks like it's part of webpack in our case?

![screen shot 2016-01-11 at 11 24 28 am](https://cloud.githubusercontent.com/assets/134455/12239284/eff06d58-b855-11e5-96be-636a2cfc91a3.png)